src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -62,6 +64,247 @@ static ShardPlacement * TupleToShardPlacement(TupleDesc tupleDesc,
 /* exports for SQL callable functions */
 PG_FUNCTION_INFO_V1(master_stage_shard_row);
 PG_FUNCTION_INFO_V1(master_stage_shard_placement_row);
+PG_FUNCTION_INFO_V1(citus_table_size);
+PG_FUNCTION_INFO_V1(citus_total_relation_size);
+PG_FUNCTION_INFO_V1(citus_relation_size);
+
+/*
+ * SupportFunctionForCitusSize is a helper function for all citus size functions.
+ *
+ */
+uint64
+SupportFunctionForCitusSize(Oid distributedRelationId, CitusSizeFunctionTypes
+							sizeFunctionType)
+{
+	DistTableCacheEntry *distributedTableCacheEntry = NULL;
+
+	/* Build shard qualified name. */
```

Comment:
In single line comments we do not start with capital letter and we do not end it with a period. We do these in multi line comments.